ktor-http/test/io/ktor/tests/http/HeadersTest.kt

```diff
@@ -181,7 +181,7 @@ class HeadersTest {
     }
 
     @Test
-    fun `headersOf should be case insensitive`() {
+    fun headersOfShouldBeCaseInsensitive() {
```

Comment:
Hopefully this would be supported in the future in all the backends or at least in none. But in js it is possible using `obj[“my property”]`